src/Avalonia.Controls/DropDown.cs

```diff
@@ -232,6 +258,22 @@ private void PopupClosed(object sender, EventArgs e)
         private void PopupOpened(object sender, EventArgs e)
         {
             TryFocusSelectedItem();
+
+            _subscriptionsOnOpen?.Dispose();
+            _subscriptionsOnOpen = null;
+
+            var toplevel = this.GetVisualRoot() as TopLevel;
+            if (toplevel != null)
+            {
+                _subscriptionsOnOpen = toplevel.AddHandler(PointerWheelChangedEvent, (s, ev) =>
```

Comment:
Good catch! Actually though I'm not sure that this is the right place to eat these scroll events. Is there ever a case where a scroll event from a popup should be passed to the parent? If so, this logic should probably be move to `PopupRoot`?

Reply:
this logic is more or less specific to dropdown so better stay in dropdown not in popup or in popuproot. As Popup can be used in another scenarios, that mouse scroll is needed for underlying window.
in Wpf if you use Popup.IsOpen over a list box and scroll with mouse listbox items are scrolled so scroll event is not swallowed.
But in wpf scroll is handled for dropdown when it's open, hence the pr logic

Reply:
Ok, that makes sense. Lets get this merged then!